### DISCUSSION TITLE
Can someone help the novice

### DISCUSSION THREAD

**Discussion by hazelldrums** [tasmota/tasmotizer#45] [Q&A] [answered]
I've installed python on my mac and run pip3 install tasmotizer on my Terminal.

But i get this screen when trying to open the program

tvfox:~ michaelhazell$ tasmotizer.py
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/bin/tasmotizer.py", line 7, in <module>
    import tasmotizer_esptool as esptool
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/tasmotizer_esptool.py", line 36, in <module>
    from PyQt5.QtCore import pyqtSignal, QObject, pyqtSlot
ImportError: dlopen(/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/PyQt5/QtCore.abi3.so, 2): Symbol not found: _LSCopyDefaultApplicationURLForURL
  Referenced from: /Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/PyQt5/Qt/lib/QtCore.framework/Versions/5/QtCore
  Expected in: /System/Library/Frameworks/CoreServices.framework/Versions/A/CoreServices
 in /Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/PyQt5/Qt/lib/QtCore.framework/Versions/5/QtCore

What am i missing here?

**Comment by MPW1412** [comment]
Hi,

looks like you didn't install PyQt5. And you'll also need pyserial.

And in my case, I also had to install libxcb-xinerama0-dev

```
sudo pip3 install PyQt5 pyserial
sudo apt-get install libxcb-xinerama0-dev
```

**Comment by feliperafael** [comment, marked as answer]
I had the same problem. I decided to install by "conda"

look at this issue: 
#https://github.com/tasmota/tasmotizer/issues/63
